group_vars/all/all.yml

```diff
@@ -38,8 +38,9 @@ helm_arch: amd64
 
 # SOLR
 # Based on https://hub.docker.com/_/solr/
-solr_image: "ckan/ckan-solr-dev"
-solr_image_tag: "2.9-py2"
+solr_image: "ckan/ckan-solr"
+solr_image_tag: "2.10"
```

Comment:
@mixmixmix That should be also freezed in WHO Romania group_vars so it's not inadvertently updated by changes done in `all.yml`.

Reply:
Why? Isn't the whole point that here it is set for `all`? And we talk here about `inventory/group_vars/wrc_prod` right?

Reply:
@tomeksabala as far as I understand if it is set up here, then it will be also updated to 2.10 in Romania which is what we want. Unless there is a good reason for Romania to use old version of solr, this is fine.